.github/workflows/elixir.yml

```diff
@@ -29,7 +29,7 @@ jobs:
             --health-retries 5
 
       clickhouse:
-        image: yandex/clickhouse-server:21.11
+        image: clickhouse/clickhouse-server:22.6
```

Comment:
- [ ] does v21.11 not support params? https://clickhouse.com/docs/en/interfaces/http/#cli-queries-with-parameters

Reply:
Seems to be just `Identifier`:

- https://github.com/ClickHouse/ClickHouse/issues/27226
- https://github.com/ClickHouse/ClickHouse/pull/28668

I'll update chto to not use it for now.